recipe/0064-opera-one-canvas/manifest.json

```diff
@@ -0,0 +1,122 @@
+{
+  "@context": "http://iiif.io/api/presentation/3/context.json",
+  "id": "{{ id.url }}",
+  "type": "Manifest",
+  "label": { "it": [ "L'elisir d'amore" ], "en": [ "The elixir of love" ] },
+  "metadata": [
+    {
+      "label": { "en": [ "Date Issued" ] },
+      "value": { "en": [ "2019" ] }
+    },
+    {
+      "label": { "en": [ "Publisher" ] },
+      "value": { "en": [ "Indiana University Jacobs School of Music" ] }
+    }
+  ],
+  "items": [
+    {
+      "id": "{{ id.path }}/canvas/1",
+      "type": "Canvas",
+      "width": 1920,
+      "height": 1080,
+      "duration": 7278.422,
+      "items": [
+        {
+          "id": "{{ id.path }}/canvas/1/annotation_page/1",
+          "type": "AnnotationPage",
+          "items": [
+            {
+              "id": "{{ id.path }}/canvas/1/annotation_page/1/annotation/1",
+              "type": "Annotation",
+              "motivation": "painting",
+              "target": "{{ id.path }}/canvas/1",
+              "body": {
+                "id": "https://fixtures.iiif.io/video/indiana/donizetti-elixir/vae0637_accessH264_low.mp4",
+                "type": "Video",
+                "format": "video/mp4",
+                "height": 1080,
+                "width": 1920,
+                "duration": 3971.24
+              }
+            }
+          ]
+        },
+        {
+          "id": "{{ id.path }}/canvas/1/annotation_page/1",
+          "type": "AnnotationPage",
+          "items": [
+            {
+              "id": "{{ id.path }}/canvas/1/annotation_page/1/annotation/2",
+              "type": "Annotation",
+              "motivation": "painting",
+              "target": "{{ id.path }}/canvas/1",
+              "body": {
+                "id": "https://fixtures.iiif.io/video/indiana/donizetti-elixir/vae0637_accessH264_low_act_2.mp4",
+                "type": "Video",
+                "format": "video/mp4",
+                "height": 1080,
+                "width": 1920,
+                "duration": 3307.22
+              }
+            }
+          ]
+        }
+      ],
+      "thumbnail": [
+        {
+          "id": "https://media.dlib.indiana.edu/master_files/vt150p78b/thumbnail",
```

Comment:
This thumbnail image (and the ones in 0065) should get uploaded and served from fixtures.iiif.io, right?

Reply:
Good point. It would make sense to and include it in the same directory as the video but it will complicate my redirecting but I think its the right thing to do. 

Do you have access to the s3 bucket or would you like me to upload it?